src/events.js

```diff
@@ -0,0 +1,175 @@
+var EventTypes = {
```

Comment:
Since we're encapsulating such minute chunks of code, can we be aggressive with the nomenclature? In this case can we just call this variable `Types`?

Reply:
I'll just inline this.